app/design/adminhtml/default/openmage/layout/local.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<layout version="0.1.0">
+    <default>
+        <reference name="head">
+            <action method="addLinkRel"><rel>stylesheet</rel><href>//fonts.googleapis.com/css?family=Open+Sans:400,300,600,700</href></action>
```

Comment:
Why is it necessary to load fonts from remote?

Reply:
good point, i forgot to address this - was somewhere on my todo list. The Open Sans Font could be dropped, or alternatively - what i would prefer - distributed with openmage. It is licensed under the Apache License which permits distribution as long as a copy of the license is packed along.
I like the lighter font weight that is possible with this font, so i would not drop it to fall back to Arial...

Reply:
i have added the Open Sans Font as local font to theme and removed the link to google fonts. I hope that embedding the font to OpenMage would be acceptable. I have added the license.txt for the font too.